emission/tests/storageTests/TestTimeSeries.py

```diff
@@ -36,10 +36,12 @@ def setUp(self):
         etc.setupRealExample(self, "emission/tests/data/real_examples/shankari_2015-aug-27")
 
     def tearDown(self):
+        edb.get_timeseries_db().delete_many({"user_id": self.testUUID1})
         edb.get_timeseries_db().delete_many({"user_id": self.testUUID})
+        edb.get_analysis_timeseries_db().delete_many({"user_id": self.testUUID1})
+        edb.get_analysis_timeseries_db().delete_many({"user_id": self.testUUID})
```

Comment:
Good job cleaning this up. I am not going to insist on it this time, but in the future, you should really put these unrelated changes into a separate commit to keep the commit history clean. You can have multiple commits in a PR, and having separate commits will keep the "git blame" more relevant.